secrets.sh

```diff
@@ -247,12 +247,12 @@ enc() {
     fi
 }
 
-# Name references ("declare -n" and "local -n") are a Bash 4 feature.
+# Name references ("declare -n" and "local -n") are a Bash 4.3 feature.
 # For previous versions, work around using eval.
 decrypt_helper() {
     local yml="$1" __ymldec __dec
 
-    if [[ ${BASH_VERSINFO[0]} -lt 4 ]]
+    if [[ ${BASH_VERSINFO[0]} -lt 4 || ${BASH_VERSINFO[0]} -eq 4 && ${BASH_VERSINFO[1]} -lt 3 ]]
```

Comment:
Hi @kincl Unfortunately you need to make this change on line 283 below as well.

Reply:
Ah good catch, done!